js/bitget.js

```diff
@@ -3897,15 +3962,32 @@ module.exports = class bitget extends Exchange {
         //        "data": "SUCCESS"
         //    }
         //
-        const timestamp = this.safeInteger (transfer, 'requestTime');
+        // fetchTransfers
+        //
+        //     {
+        //         "cTime":"1622697148",
+        //         "coinId":"22",
+        //         "coinName":"usdt",
+        //         "groupType":"deposit",
+        //         "bizType":"transfer-in",
+        //         "quantity":"1",
+        //         "balance": "1",
+        //         "fees":"0",
+        //         "billId":"1291"
+        //     }
+        //
+        let timestamp = this.safeInteger (transfer, 'requestTime');
```

Comment:
something is off here
![image](https://user-images.githubusercontent.com/43336371/215520864-4485dd2d-2bff-4a1a-98ec-9580ec265a72.png)